setuptools/tests/environment.py

```diff
@@ -18,6 +18,13 @@ class VirtualEnv(jaraco.envs.VirtualEnv):
     def run(self, cmd, *args, **kwargs):
         cmd = [self.exe(cmd[0])] + cmd[1:]
         kwargs = {"cwd": self.root, **kwargs}  # Allow overriding
+        # If no environment was passed as an argument
+        # create a copy of the current env to use when spawning a subprocess
+        # Unset PYTHONPATH - tests in venv need only that venv's paths in sys.path
+        env = kwargs.get("env") or dict(os.environ)
```

Comment:
Is it possible to not touch an explicitly given `env` here? In some scenarios tests might explicitly set `PYTHONPATH`.